src/main/scala/com/github.sstone/amqp/ChannelOwner.scala

```diff
@@ -25,13 +28,34 @@ object ChannelOwner {
 
   private[amqp] class Forwarder(channel: Channel) extends Actor with ActorLogging {
 
+    private var uuid: Option[UUID] = None
```

Comment:
Probably this shouldn't be an Option, instead we should initialize this when creating the new forwarder instance.